ecommerce/templates/checkout_interstitial.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
Could you have this view extend the base template as well so that the user doesn't see an unstyled page?

Reply:
Fixed in last commit https://github.com/mitodl/mitxonline/pull/418/commits/2f86fae21aa974018012cf31ceca94116eb24f2f

This won't actually perform the redirect (the code for that is commented out) so it's easier to review.